src/Surfnet/StepupMiddleware/ApiBundle/Identity/Repository/RaCandidateRepository.php

```diff
@@ -69,13 +69,34 @@ public function merge(RaCandidate $raCandidate)
      */
     public function removeByIdentityId(IdentityId $identityId)
     {
-        $raCandidate = $this->findByIdentityId($identityId);
+        $raCandidates = $this->findByIdentityId($identityId);
 
-        if (!$raCandidate) {
+        if (count($raCandidates) < 1) {
```

Comment:
Prefer !$raCandidates or empty($raCandidates)

Reply:
Po-tay-to po-tah-to, but changed.